backend/package.json

```diff
@@ -17,7 +17,7 @@
     "prestart": "npm run migrate:prod && npm run seed:soft",
     "start": "node ./build/server.js",
     "build:ts": "tsc && tsc-alias -p tsconfig.json",
-    "build": "npm run build:ts"
+    "build": "npm run build:ts && cp -r src/shared/mail-templates build/shared/mail-templates"
```

Comment:
It seems better to execute this command during building the shared package. Also, could you please create a js file (`build.js`) and implement copy logic in the js file using the `fs` module from `nodejs`?

Reply:
so should I move mail templates from backend to shared folder as well?